cmd/bosun/web/chart.go

```diff
@@ -122,7 +122,7 @@ func Graph(t miniprofiler.Timer, w http.ResponseWriter, r *http.Request) (interf
 	}
 	if _, present := r.Form["png"]; present {
 		c := chart.ScatterChart{
-			Title: fmt.Sprintf("%v - %v", oreq.Start, oreq.End),
+			Title: fmt.Sprintf("%v - %v", oreq.Start, queries),
```

Comment:
End was often nil and it wasn't handled. I'd vote for yanking out this feature all together.